Add agenda doc details and schedule skeleton (#139)
nodejs/summit: 2019-05-Berlin/agenda.md

# Collaborator Summit Berlin 2019 - May 30th and 31th 🇩🇪

This is a live document that will be constantly updated to match the proposed agenda.

> Our time will be best served with some monotrack presentations and discussions in the first day, and then breakout rooms in the second day. __We can change everything__, so if you think a better structure will suit us better, let us know in #135!

Reviewer: Based on some of the feedbacks from Vancouver, we are best using our time if the sessions have quorum:

```suggestion
A collaborator summit is a mix of presentations and interactive sessions lead by the Teams, Working Groups, and Projects that participate in the summit. For interactive sessions, our time is best served if a the team/working groups/project participate in good numbers (quorum etc).

__We can change everything__, so if you think a better structure will suit us better, let us know in #135, or just send a PR!
```

Author: The specificity on which "entities" can lead the sessions is a solid improvement on the earlier language 👍🏽

Totally! Setting a level for quorum ahead of time is a clear way to find out whether the session will be of value. Can the proposals feasibly estimate the minimum numbers for their session to activate? If so, we can ask them to.

@joyeecheung did a rad survey of which sessions people would attend. This time it would be great to more deeply integrate that into the numbers planning!

> It would be fantastic if each one of the breakouts is facilitated by a person who will be responsible to "run" the breakout and do a quick presentation at the end. This person would also set outcomes from the breakout to  optimally leverage our time together.

Reviewer: ```suggestion
It would be fantastic if each one of the breakouts is facilitated by a person who will be responsible to "run" the breakout and do a quick presentation at the end. This person would also set outcomes from the breakout to  optimally leverage our time together. There is a facilitator guide available [here](https://github.com/nodejs/summit/blob/master/SESSION_FACILITATOR_GUIDE.md).
```

✨ We're excited for Node.js' 5th round of collaboration IRL! Even better, we're excited that folks from the JSF will be joining us for the first time! ✨

> __Location TBA__,
(Street Address)
Berlin, Germany

## Collaborator Summit Day One: 30th May, 2019

(Time window)

### Agenda

(Types of rooms available):

* (details of main room)
* (details of more rooms)

_(Expected participation number)_.

> | Time          | Place        | Topic                                         | Speakers/Champions                     | Issue                                      | Slides                                    |

Reviewer: ```suggestion
| Time          | Place        | Topic                                         | Speakers/Facilitators                     | Issue                                      | Slides                                    |
```

|---------------|--------------|-----------------------------------------------|----------------------------------------|--------------------------------------------|-------------------------------------------|
| | Main         | Intro, Sponsor Talk, Code of Conduct          | |                                            | |

_To speak/present a topic/do a sesssion, make a pull request to update this document with a title, abstract, and a link to an issue with more detail in this repo, similar to https://github.com/nodejs/summit/issues/86_.

## Collaborator Summit Day Two: 31st May, 2019

(time window)

### Agenda

(Types of rooms available):

* (details of main room)
* (details of more rooms)

_(Expected participation number)_.

> | Time          | Place        | Topic                                                               | Speakers/Champions            | Issue                                      | Slides                                    |

Reviewer: ```suggestion
| Time          | Place        | Topic                                                               | Speakers/Facilitators            | Issue                                      | Slides                                    |
```

|---------------|--------------|---------------------------------------------------------------------|-------------------------------|--------------------------------------------|-------------------------------------------|
| | Main | Intro, Code of Conduct                                              | |                                            | |

_To speak/present a topic/do a sesssion, make a pull request to update this document with a topic, contact @github-handle. Include a link to an issue in this repo with more detail, similar to https://github.com/nodejs/summit/issues/86_ that contains at least an abstract, a link to the planning issue in the working group repo, and any extra requests (minimum space during conf, av/sound, etc.).

## Organizers

> - name (@github-handle)

Reviewer: ```suggestion
- Matteo Collina (@mcollina)
```

(But there are also a lot of other people, including yourself ;))

Author: Right on, I'll add myself ;) and everyone who's expressed willingness in the issue

> 

Reviewer: I would like to add a new section about "Code of Conduct" info & contacts for the event.

Author: Please do feel free to commit to the branch! I'm not sure how much content and which contacts you'd like to make available 🙏🏽

Author: Alternatively, that could be a new issue to be addressed following this PR being merged.

Author: Resolving this and creating #140.

Reviewer 2: > I would like to add a new section about "Code of Conduct" info & contacts for the event.

It would be awesome @mcollina! I agree with you.

## Organizers

- Matteo Collina (@mcollina)
- Manil Chowdhury (@chowdhurian)
-  (@trivikr)
-  (@thefourtheye)
-  (@mmarchini)
-  (@sagirk)
-  (@WaleedAshraf)
-  (@joyeecheung)
-  (@jorydotcom)

## Attendance

Anyone can come, but we will not be explicitly onboarding at Collab 
Summit Vancouver. Conversations can move fast as working groups have 
a lot of context. Unlike the Fall instance of the Collab Summit, this Spring instance does not include a morning of Code + Learn, the session that guides people through contributing to the project.

Working groups will put together a brief schedule so that people can 
familiarize themselves before folks get onsite, having the general collaborator 
discussions, and then dive into breakout sessions.

Attendees must register at: (TBA)

  
## Travel Fund

> There are funds available for any Individual Member of the Node.js Foundation to receive travel funds to Collab Summit.  Membership is free for active collaborators. Sign up [HERE](https://github.com/nodejs/members#members).

Reviewer 3: @chowdhurian Links seems to 404 here

Reviewer 4: @mbj36 This is a private repo, so 404 is expected.

Reviewer 3: Thanks for clarifying
